types/react-star-ratings/index.d.ts

```diff
@@ -0,0 +1,29 @@
+// Type definitions for react-star-ratings 2.3
+// Project: https://github.com/ekeric13/react-star-ratings
+// Definitions by: Andrew Shkurenko <https://github.com/vanillkay>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.8
+
+import * as React from "react";
+
+interface StarRatingProps {
+    rating?: number;
```

Comment:
```suggestion
    rating: number;
```
is this one required? https://github.com/ekeric13/react-star-ratings/blob/07add0e61bf77fb94a9613587e31bffa9f46364e/src/star-ratings.js#L180

Reply:
same for some others

Reply:
never mind, read the defualts